config/templates/cloud-config-controller

```diff
@@ -693,6 +706,66 @@ write_files:
            $kubectl uncordon "$hostname"
            echo "done."'
 
+  {{if .Experimental.Plugins.Rbac.Enabled }}
+
+  #https://github.com/coreos/kube-aws/issues/230#issuecomment-274946933
+  - path: /srv/kubernetes/rbac/clusterrole.yaml
```

Comment:
nit but how about organizing and naming yaml files after their kinds and `metadata.name` fields like:

* /srv/kubernetes/rbac
  * cluster-roles/
     * cluster-admin.yaml
  * cluster-role-bindings/
     * kube-admin.yaml 
     * kube-worker.yaml
     * system-default.yaml